stable/concourse/README.md

```diff
@@ -382,6 +382,8 @@ concourse:
 
 To use SSM, set `concourse.web.kubernetes.enabled` to false, and set `concourse.web.awsSsm.enabled` to true.
 
+Authentication can be configured to use an access key and secret key as well as a session token. This is done by setting `concourse.web.awsSsm.keyAuth.enabled` to `true`. Alternatively, if it set to `false, AWS IAM role based authentication (instance or pod credentials) is assumed. To use a session token, `concourse.web.awsSsm.useSessionToken` should be set to `true`. The secret values can be managed using the values specified in this helm chart or separately. For more details, see https://concourse-ci.org/creds.html#ssm.
```

Comment:
Hey, sorry for noticing it just now, but after "`Alternatively, if it set to`", `false` is not being closed with "`", messing up with the formatting for the rest of the paragraph.

Reply:
Good find! Better sorted now than later. I've added a commit to fix it.